native/src/main/java/io/ballerina/stdlib/http/api/HttpConnectionManager.java

```diff
@@ -143,7 +143,7 @@ public boolean isHTTPTraceLoggerEnabled() {
         return Boolean.parseBoolean(System.getProperty(HttpConstants.HTTP_TRACE_LOG_ENABLED));
     }
 
-    private boolean isHTTPAccessLoggerEnabled() {
+    public boolean isHTTPAccessLoggerEnabled() {
```

Comment:
When we set a system property, it affects the whole jvm right? If there are 2 services where 1 service has accesslog enabled and other one doesn't, this will confuse the behaviour right?